packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -967,15 +989,19 @@ export class GarbageCollector implements IGarbageCollector {
 		}
 
 		const errorRequest: IRequest = request ?? { url: nodePath };
-		// If the object is tombstoned and tombstone enforcement is configured, throw an error.
-		if (isTombstoned && this.throwOnTombstoneLoad && headerData?.allowTombstone !== true) {
-			// The requested data store is removed by gc. Create a 404 gc response exception.
-			throw responseToException(
-				createResponseError(404, `${nodeType} was tombstoned`, errorRequest, {
-					[TombstoneResponseHeaderKey]: true,
-				}),
-				errorRequest,
-			);
+		if (isTombstoned) {
+			this.submitTombstoneLoadedMessage(nodePath);
```

Comment:
I was thinking that the summarizer client should not submit this message but directly call `this.addedOutboundReference`. The reason being that this can result in docs becoming unrecoverable where there are 10k unacked ops. If the summarizer client submits this op before or after the summary that would recover the document, this will be nacked and the summarizer will close.